### PR TITLE
Lektion 1

### PR DESCRIPTION
**Pull request by derfabio**
Pull Request für Übung 1 und Übung 2 der Lektion 1 zum Thema "Was ist Semantisches HTML?"

### CODE REVIEW

**Review by Hanmuni — APPROVED** [review]

File: derFabio/Übung2.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Webshop Animal Farm</title>
+</head>
+<body>
+    <header>
+        <h1>Welcome to the Animal Farm Webshop</h1>
+    </header>
+    <div class="container">
```
Comment (Hanmuni) [inline derFabio/Übung2.html]: Why do you wrap the navbar in a <div> when <nav> already shows what it does?

**Review by Hanmuni — CHANGES_REQUESTED** [review]
I think it's already very good. Look for my comment in line 12 for minor changes.

File: derFabio/Übung2.html (hunk 1 above)
Comment (derfabio) [reply derFabio/Übung2.html]: My thinking was to have the navbar as part of the container div for the main content of the page but maybe the div might not be needed altogether.

Comment (hola-soy-milk) [reply derFabio/Übung2.html]: Thanks for your review! I find @derfabio's reply to be solid here. Wrapping the non-header content in a container div makes sense.

I'd even go as far as to say this showcases the best use of divs: for division!

What do you think, @Hanmuni ?

File: derFabio/Lektion03: Gründe für semantisches HTML: SEO/Übung1 [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Artikelvorschau mit Rich Snippets:
+https://asana.com/de/resources/time-management-tips
+
+Durch Rich Result Test können drei valide items detected werden und crawling ist erlaubt.
+Die Seite könnte verbessert werden durch Verwendung von Bildern, die Gegenstände zur Organisation zeigen. Beispiele dafür könnten sein: Terminkalender, To-Do Liste und Stift, Eieruhr (Pomodoro Technik).
```
Comment (hola-soy-milk) [inline derFabio/Lektion03: Gründe für semantisches HTML: SEO/Übung1:5]: Great work!

**Review by hola-soy-milk — CHANGES_REQUESTED** [review]
Thanks for this! I noticed that a lot of files here don't have an extension. Is it possible these were meant to be markdown? 

Maybe they need to have backticks aroud the html?

File: derFabio/Übung2.html (hunk 1 above)
Comment (Hanmuni) [reply derFabio/Übung2.html]: In my opinion, it's more understandable to relocate the navbar to the header section. When you first open the browser, in my opinion, this structure is better to understand:

   ```
<header>
        <h1>Welcome to the Animal Farm Webshop</h1>
        <nav></nav>
    </header>

    <main>
        <section></section>

        <section></section>

        <aside></aside>
    </main>

    <footer></footer>
```

Having the <main> tag in your browser lets the user know that the main content is in there. A <div class="container"> could mean anything.

Comment (hola-soy-milk) [reply derFabio/Übung2.html]: Ahhh! That makes lots of sense. What do you think, @derfabio ?

Comment (derfabio) [reply derFabio/Übung2.html]: I think @Hanmuni 's way of structuring the page makes a lot of sense.
It gives a clean view of what is going on in the page, I like it 👍🏻

**Comment by hola-soy-milk** [comment]
Great work!! 🥳 

Thank you @Hanmuni for your contributions!!

**Pull request merged**